streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -877,7 +877,7 @@
         public static final String STREAMS_METADATA_STATE_FOR_PARTITION_ASSIGNOR = "__streams.metadata.state.instance__";
         public static final String STREAMS_ADMIN_CLIENT = "__streams.admin.client.instance__";
         public static final String ASSIGNMENT_ERROR_CODE = "__assignment.error.code__";
-        public static final String NEXT_PROBING_REBALANCE_MS = "__next.probing.rebalance.ms__";
+        public static final String NEXT_SCHEDULED_REBALANCE_MS = "__next.probing.rebalance.ms__";
```

Comment:
Minor detail, but we reserve the term "probing rebalance" for the KIP-441 type of rebalances, while now this is used to schedule any kind of followup rebalance

Reply:
Did you want to change the config key as well? I think this should be ok, since it's only used to pass state from one object to another within the same instance, so there should be no compatibility concern.

Reply:
Are you asking if this change was intentional, or are you referring to a different config key? If the former, note that this internal config was only added a few PRs ago so there would be no compatibility concerns even if it wasn't confined to a single instance

Reply:
```suggestion
        public static final String NEXT_SCHEDULED_REBALANCE_MS = "__next.scheduled.rebalance.ms__";
```

That's what I meant (just remembered I can use the change recommendation thingy).

Reply:
It's a very minor question, I only ask because you were trying to make everything consistent here.

Reply:
Oh, duh, I overlooked that. Guess I was just going for eventual consistency here